Makefile

```diff
@@ -43,6 +43,8 @@ install:
 	mkdir -p $(DESTDIR)$(bindir)
 	$(INSTALL_DATA) bleachbit.py $(DESTDIR)$(bindir)/bleachbit
 	chmod 0755 $(DESTDIR)$(bindir)/bleachbit
+	# Update the hardcoded /usr/share in bleachbit.py to match the datadir set by the user.
+	sed -i 's|/usr/share|$(datadir)|g' $(DESTDIR)$(bindir)/bleachbit
```

Comment:
@Zopolis4 
There are many references to `/usr/share` in `__init__.py`, though not all should be changed. One solution is to make a Python variable with `/usr/share` that sed can modify.

`/usr/share/common-licenses/GPL-3` is an example that should not be modified.

Reply:
I'm only modifying `bleachbit.py` here, unless you mean there are further portability problems in `__init__.py` that I have not adressed?

Reply:
@Zopolis4 
Yes, search `__init__.py` for `/usr/share`.